File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/baseClass/baseCodeDescrStr.ts
import {BaseCodeDescrStr as IBaseCodeDescrStr} from "../../Service/api.service";

export class BaseCodeDescrStr implements IBaseCodeDescrStr {
    codice: string;
    descrizione: string;

    constructor(codice: string, descrizione?: string) {
        this.codice = codice;
        this.descrizione = descrizione;
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/baseClass/BaseCodeDescrStr.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.baseClass
{
    public class BaseCodeDescrStr
    {
        public string codice { get; set; }
        public string descrizione { get; set; }
        public BaseCodeDescrStr(string code, string description)
        {
            this.codice = code;
            this.descrizione = description;
        }

        public BaseCodeDescrStr()
        {
            this.codice = "";
            this.descrizione = "";
        }

    }
}
